src/main/java/data/Timeline.java

```diff
@@ -0,0 +1,33 @@
+package data;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by Bart.
+ * Abstract class for timelines.
+ */
+public abstract class Timeline {
+
+    // Description of this timeline
+    @Getter
+    private String description;
+
+    // Lowest count of this timeline
+    @Getter @Setter
+    private int beginCount;
```

Comment:
The stakeholders purely talked about a count number that they were using to point out time, this can easily be modelled with an integer. I don't see any other things we would need from a time oriented class.